clients/openshift.go

```diff
@@ -74,7 +74,7 @@ func newOpenshift() (*OpenshiftClient, error) {
 	// library
 	clientConfig, err := rest.InClusterConfig()
 	if err != nil {
-		log.Warningf("Failed to create a InternalClientSet: %v.", err)
+		log.Debugf("Failed to create a InternalClientSet: %v.", err)
```

Comment:
Same deal here, line 83 should handle printing a user-visible error message, so my take is that this log statement being a "Warning" isn't critical

Reply:
I would advocate to just remove these message altogether. Because like you said line 83 will print the real error. I don't like seeing a statement "Failed to create " as debug.